packages/zoe/src/zoe.js

```diff
@@ -334,8 +334,12 @@ const makeZoe = (additionalEndowments = {}) => {
     offerTable,
     payoutMap,
     issuerTable,
+    brandToIssuerMap,
   } = makeTables();
 
+  const getAmountMathForBrand = brand =>
+    issuerTable.get(brandToIssuerMap.get(brand)).amountMath;
```

Comment:
some callers that use issuerTable have brands, some have issuers. The main datum that contracts usually start from is the IssuerKeywordRecord. If we want to change that to a BrandKeywordRecord, I think this would make sense, otherwise I think it makes more work internally, and more work for contracts.

What makes you suspect it would be beneficial?

Reply:
Hmm, I don't get why the existence of an issuerKeywordRecord in the instanceTable would mean we need the issuer to be the key for the issuerTable. Zoe was built before we had brands in ERTP, so the fact that the issuer is the key for the table is just legacy.  Brands should really be the identifier for issuers/purses/amountMath.

Reply:
I converted the table to be keyed by brand, but added a function to get the brand if you have the issuer.